webpack/module-minifier-plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./node_modules/@microsoft/rush-stack-compiler-3.5/includes/tsconfig-node.json",
+  "compilerOptions": {
+    "strictNullChecks": true,
+    "lib": ["ES2017"],
+    "target": "ESNext",
```

Comment:
What is `es2017` missing?

Reply:
`import()`, though I'm not currently using it.